### ISSUE TITLE
Link to Discount edit screen from payment record

### ISSUE THREAD

**Issue by SDavisMedia** [easydigitaldownloads/easy-digital-downloads#5325]
It would be great if the discount code(s) on the payment record were linked to their edit screens, as the record is a hub for everything else related to the payment.

<img width="307" alt="screenshot 2016-12-29 10 47 43" src="https://cloud.githubusercontent.com/assets/2359131/21552128/6203a422-cdb4-11e6-9387-a2a57672f92f.png">

If everyone agrees, this would be a great addition once `EDD_Discount` is ready. https://github.com/easydigitaldownloads/easy-digital-downloads/issues/5217

**Comment by sunnyratilal** [comment]
Yep, it's something small but a nice addition at the same time. We'll write the code up once #5217 is merged.

**Comment by sunnyratilal** [comment]
In #5217, I've added an `edit_url()` function to make this even easier.

**Comment by sunnyratilal** [comment]
Commit ref: 30c42bc7e817e7b0cfd80b3ad7fc9b78ffc0a1c8

**Comment by cklosowski** [comment]
Reopening for some visual updates.

**Comment by sunnyratilal** [comment]
PR tested and I'm happy with it 👍

**Comment by SDavisMedia** [comment]
Beautiful. Thumbs up from me as well. :+1:

**Comment by sunnyratilal** [comment]
Merged into `release/2.7`

**Comment by pippinsplugins** [comment]
Very nice!

**Comment by ghost1227** [comment]
@SDavisMedia Improved discount list ftw!

<img width="288" alt="screen shot 2017-01-17 at 10 24 35 am" src="https://cloud.githubusercontent.com/assets/36433/22029142/398d81b2-dc9f-11e6-8d3a-96f4b26bc5c0.png">
